projects/dotcms-js/src/lib/core/core-web.service.ts

```diff
@@ -135,77 +150,104 @@ export class CoreWebService {
      */
     public requestView(options: RequestOptionsArgs): Observable<ResponseView> {
         const request = this.getRequestOpts(options);
-
-        return this._http.request(request).pipe(
-            map((resp) => {
-                if (resp.json().errors && resp.json().errors.length > 0) {
+        return this.http.request(request).pipe(
+            filter(<T>(event: HttpEvent<T>) => event.type === HttpEventType.Response),
```

Comment:
What is this filter for? and where are those `<T>` coming from?

Reply:
https://github.com/dotCMS/core-web/pull/1429/files#r493914633

Reply:
This link don't show me anything in particular.